js/main.js

```diff
@@ -48,6 +48,32 @@ jQuery(document).on('ready', function() {
 			scrollTop: jQuery("#tg-main").offset().top + offset
 		}, 2000);
 	});
+
```

Comment:
@CuriousLearner I earlier submitted a PR #50, whose code remained in this branch when I checkout naming the new branch `travis-ci`, that's why the code is appearing here.
- Once the PR#50 is merged, it will not show these changes. 
- I will remember to keep the `master` branch clean onwards